Lib/fontTools/feaLib/parser.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, featurefile, glyphNames=(), followIncludes=True,
         lexerClass = IncludingLexer if followIncludes else NonIncludingLexer
         self.lexer_ = lexerClass(featurefile)
         self.advance_lexer_(comments=True)
+        self.SS_FEATURE_TAGS = ["ss%02d" % i for i in range(1, 20+1)]
+        self.CV_FEATURE_TAGS = ["cv%02d" % i for i in range(1, 99+1)]
```

Comment:
I actually meant class attributes, as they are constant. Not instance attributes initialised in the constructor.

Reply:
I originally placed them just below line #18 `ast = ast` and it failed. I must have missed something when I tried that.